### DISCUSSION TITLE
my app freezes

### DISCUSSION THREAD

**Discussion by Giazzo78** [AndreyPavlenko/Fermata#81] [Q&A]
my app freezes after 5 minutes of use, how can I fix. Thank you

**Comment by AndreyPavlenko** [comment]
Could you identify the action that causes freeze? Play video, click on a button etc.

**Comment by Giazzo78** [comment]
Non c’è una specifica azione che fa disconnettere tutto, ma dopo 5 minuti si interrompe qualunque riproduzione.
——-

There is no specific action that disconnects everything, but after 5 minutes, any playback stops.
———-

Inviato da iPhone
